VideoWeb/VideoWeb.AcceptanceTests/Steps/JudgePrivateConsultationSteps.cs

```diff
@@ -86,7 +86,7 @@ public void WhenTheyEnterPrivateConsultationRoom()
         {
             _browsers[_c.CurrentUser].Driver.WaitUntilVisible(JudgeWaitingRoomPage.HearingTitle).Displayed.Should().BeTrue();
             WaitForUserStatusToBe(ParticipantState.Available);
-            _browsers[_c.CurrentUser].ClickToProgress(JudgeWaitingRoomPage.EnterPrivateConsultationButton, PrivateConsultationRoomPage.LeavePrivateConsultationButton, 10);
+            _browsers[_c.CurrentUser].ClickToProgress(JudgeWaitingRoomPage.EnterPrivateConsultationButton, PrivateConsultationRoomPage.LeavePrivateConsultationButton);
```

Comment:
Has this been tested in a pipeline?
There should have been a reason for setting the timeout override.